docs/src/config/sidebars.ts

```diff
@@ -3301,6 +3301,11 @@ const sidebars: SidebarsConfig = {
               label: 'site add',
               id: 'cmd/spo/site/site-add'
             },
+            {
+              type: 'doc',
+              label: 'site admin add',
+              id: 'cmd/spo/site/site-admin-add'
+            },
```

Comment:
Not really your mistake, but we have decided earlier that we will add command groups at the bottom of the list. Let's reorder this one under spo site set, and let's also include spo admin list

Reply:
ok I have moved it after 'site set'